packages/common/container-definitions/src/runtime.ts

```diff
@@ -184,6 +184,9 @@ export interface IContainerContext extends IDisposable {
 	getLoadedFromVersion(): IVersion | undefined;
 
 	updateDirtyContainerState(dirty: boolean): void;
+
+	readonly supportedFeatures?: ReadonlyMap<string, unknown>;
```

Comment:
(Mostly talking aloud)
I believe it's right approach, but I do not know how to think about documentation here.
I.e. we should fully support parallel implementations of IRuntime, or vice versa - IContainerContext, including by 3rd parties. It's not very clear what needs to be done here RE properly documenting all the properties we are exposing here, as their timeframe (i.e. I totally expect exposure of features to be "sunset" once we have 100% saturation of new bits, i.e. the need to check for feature presents to go away)